ckcc/cli.py

```diff
@@ -295,7 +295,7 @@ def file_upload(filename, blksize, multisig=False, miniscript=False):
         if multisig:
             dev.send_recv(CCProtocolPacker.multisig_enroll(file_len, sha))
```

Comment:
needs same change?

Reply:
nope - parsing miniscript is more complex and takes more time than just basic `sortedmulti(`. I have tested 15of15 multisig (CC max limit) and never experienced timeout